### DISCUSSION TITLE
Keybindings displayed adjacent to commands

### DISCUSSION THREAD

**Discussion by bound-variable** [helix-editor/helix#1634] [Ideas] [answered]
I used vim a bit, back over a decade ago. It would be great if keybindings were displayed next to the commands (at the bottom of the screen) when in command mode. It would really help with learning this editor, given the shortness of the tutorial.

Thanks for all the work. The Linux community is amazing.

Much love

**Comment by sudormrfbin** [comment]
I'm not sure I understand exactly what you mean here; command mode would be activated when you press `:`, and keybindings aren't relevant in that context. Also unlike vim there are no typable commands that can do the function of keybindings in helix (in vim there is the `:undo` command and the `u` keybind, in helix there is only `u`).

**Reply by bound-variable** [reply]
We don't use the bindings from command mode, sure. But it'd still be nice to see them there. Like in VSCode:

![20220209_10h13m05s_grim](https://user-images.githubusercontent.com/51193876/153264226-f2c8de81-94a5-43dd-bad4-1ee525760269.jpeg)

I think most (perhaps all) users of Helix are already familiar with vim-like editors. But this feature would really help those of us who aren't.

**Reply by sudormrfbin** [reply, marked as answer]
This exact feature is being worked on in #1400 and is set to be merged soon 🎉

**Reply by TornaxO7** [reply]
@Pound-Hash may I ask if you could mark your question as answered? :)
